funnel/static/robots.txt

```diff
@@ -0,0 +1,6 @@
+# www.robotstxt.org/
+# www.google.com/support/webmasters/bin/answer.py?hl=en&answer=156449
+
+User-agent: *
+Disallow: /account/
+Disallow: /login/
```

Comment:
Does this also block these URLs without a trailing slash? Both `/account` and `/login` are valid URLs.

Reply:
Updated, now it should match both `/login` and `/login/github` for instance. See: https://developers.google.com/search/reference/robots_txt